handlers/refunds_test.go

```diff
@@ -93,7 +93,7 @@ func TestUnitHandleProcessPendingRefundsWithPaymentRefunds(t *testing.T) {
 
 		HandleProcessPendingRefunds(w, req)
 
-		So(w.Code, ShouldEqual, http.StatusOK)
+		So(w.Code, ShouldNotEqual, http.StatusOK)
```

Comment:
This change doesn't look correct. Why is the test not returning StatusOK?

Reply:
Dicussed